docs/reference/upgrade.md

```diff
@@ -66,3 +66,62 @@ To do so, add the following to your configuration (relevant **only** for 0.30.0)
 cataloger:
   type: rocks
 ```
+
+## Data Migration for Version v0.49.0
```

Comment:
Reminder: as discussed F2F I had to use up this minor version on Thursday.  You'll have to bump in